src/coprocessor/metrics.rs

```diff
@@ -67,6 +68,30 @@ lazy_static! {
         "Total bytes of response body"
     )
     .unwrap();
+    pub static ref COPR_ACQUIRE_SEMAPHORE_TYPE: CoprAcquireSemaphoreTypeCounterVec =
+        register_static_int_counter_vec!(
+            CoprAcquireSemaphoreTypeCounterVec,
+            "tikv_coprocessor_acquire_semaphore_type",
+            "The acquire type of the coprocessor semaphore",
+            &["type"],
+        )
+        .unwrap();
+    pub static ref COPR_WAITING_FOR_SEMAPHORE: IntGauge = register_int_gauge!(
+        "tikv_coprocessor_waiting_for_semaphore",
+        "The number of tasks waiting for the semaphore"
+    )
+    .unwrap();
+}
+
+make_static_metric! {
+    pub label_enum AcquireSemaphoreType {
+        unacquired,
```

Comment:
Why `acquiring` is removed?

Reply:
Actually there isn't `acquring` before. `COPR_WAITING_FOR_SEMAPHORE` refers to `acquiring`.